src/commands/deploy/deploy.ts

```diff
@@ -21,23 +21,26 @@ import { getRandomHexString } from "../../utils/randomUtils";
 import * as workspaceUtil from '../../utils/workspace';
 import { cancelWebsiteValidation, validateWebSite } from '../../validateWebSite';
 import { getWorkspaceSetting, updateWorkspaceSetting } from '../../vsCodeConfig/settings';
-import { getDefaultWebAppToDeploy } from '../getDefaultWebAppToDeploy';
 import { startStreamingLogs } from '../startStreamingLogs';
-import { IDeployWizardContext } from './IDeployWizardContext';
+import { getDefaultWebAppToDeploy } from './getDefaultWebAppToDeploy';
+import { IDeployWizardContext, WebAppSource } from './IDeployWizardContext';
 
 // tslint:disable-next-line:max-func-body-length cyclomatic-complexity
 export async function deploy(context: IActionContext, confirmDeployment: boolean, target?: vscode.Uri | SiteTreeItem | undefined): Promise<void> {
+    context.telemetry.properties.deployedWithDefaultApp = 'false';
```

Comment:
Can we use the same `webAppSource` property for telemetry? It would be more consistent, easier to write a query for in telemetry, and I think less code since you'd only have to set it once after `webAppSource` has been set

Reply:
Well, webAppSource will always be nodePicker | setting | treeView, but deployedWithDefaultApp is being used to track whether or not people are using the defaultWebAppSetting including if they are using the `none` setting.

Reply:
That being said, we track if people click the "Don't show again" button, so I don't think we necessarily have to track the 'none' setting.